Zend/tests/exit/exit_statements.phpt

```diff
@@ -0,0 +1,46 @@
+--TEST--
+Using exit/die as a statement/constant
+--FILE--
+<?php
+
+const FILE_PATH = __DIR__ . '/exit_statements.inc';
+const FILE_CONTENT = <<<'TEMPLATE'
+<?php
+echo "Before FUNCTION";
+try {
+    FUNCTION;
+} catch (\Throwable $e) {
+    echo $e::class, ': ', $e->getMessage(), PHP_EOL;
+}
+
+TEMPLATE;
+
+
+$php = getenv('TEST_PHP_EXECUTABLE_ESCAPED');
+$command = $php . ' ' . escapeshellarg(FILE_PATH);
+
+foreach (['exit', 'die'] as $value) {
```

Comment:
Nit: There's just two of those functions, so possible simpler and clearer to just create and commit the files.

Reply:
Do you want me to create two `.inc` files?